### PR TITLE
Update facilitator guides

### PR DESCRIPTION
**Pull request by iramosp**
This PR:
* Clarifies video and call facilitation steps.
* Removes cohort/program specific information. Ideally these guides work for both Open Seeds and Nebula.
* Syncs the duplicated video facilitation guide (the duplicated file still exists, but the instructions are now the same. Still TBD whether to keep call and video facilitator guides in the same file, see #800).

Tagging @seunolufemi123 for content review and @Tajuddeen1, @malvikasharan for awareness. Any suggestions welcome.

## FOR CONTRIBUTOR

* [X] I have read the [CONTRIBUTING.md](https://github.com/open-life-science/open-life-science.github.io/blob/main/CONTRIBUTING.md) document

PR Type: 
* [ ] This PR adds a new blog post
* [x] This PR does something else (explain above)

 
## FOR REVIEWERS

Thanks for taking the time to review! :heart:

Here are the list of things to make sure of:
* [x] The website builds (a check will fail if not)
* [ ] All images have been added within the Pull Request and they have Alt text
* [ ] If there are paragraphs or text, the key messages are highlighted
* [ ] All internal links (within OLS website) use the [`{% link path_to_file.md %}` format](https://jekyllrb.com/docs/liquid/tags/#link)
* [x] The preview corresponds to the changes described in the Pull Request
* [x] The code is tidy and passes the linting tests

### CODE REVIEW

File: openseeds/cohort-procedures-and-templates/cohort-call-facilitator-guide.md [hunk 1]
```diff
@@ -10,108 +10,121 @@ photos:
...
+Facilitators share some of the cohort call-related management roles with the OLS team. 
 
-Please identify which week you have signed up for and who is the cohort call lead.
+Please identify which week you have signed up for and who the cohort call lead is.
 
-Our cohort calls vary in time from week to week. Please, see schedules from [our website]({{ site.url }}/openseeds/).
+Our cohort calls vary in time from week to week. Please, see the schedules from [our website]({{ site.url }}/openseeds/).
 
-For any questions or concerns, please contact Community Coordinator, Paz Bernaldo via Slack.
+For any questions or concerns, please contact the cohort coordinators via Slack or email.
```
Comment (NPDebs) [inline openseeds/cohort-procedures-and-templates/cohort-call-facilitator-guide.md:25]: Who are they, @iramosp?

Comment (NPDebs) [reply openseeds/cohort-procedures-and-templates/cohort-call-facilitator-guide.md:25]: Might we worth mentioning the key players. What do you think?

Comment (iramosp) [reply openseeds/cohort-procedures-and-templates/cohort-call-facilitator-guide.md]: Good idea, I added contact email for coordinators.

**Review by NPDebs — COMMENTED** [review]
Thanks a lot for the hardwork, @iramosp! Looks good to me. 💯

**Review by NPDebs — APPROVED** [review]

**Pull request merged**